AxonDeepSeg/segment.py

```diff
@@ -258,17 +196,14 @@ def main(argv=None):
     requiredName = ap.add_argument_group('required arguments')
 
     # Setting the arguments of the segmentation
-    requiredName.add_argument('-t', '--type', required=True, choices=['SEM','TEM', 'OM'], help='Type of acquisition to segment. \n'+
+    requiredName.add_argument('-t', '--type', required=True, choices=['SEM', 'TEM', 'BF'], help='Type of acquisition to segment. \n'+
                                                                                         'SEM: scanning electron microscopy samples. \n'+
-                                                                                        'TEM: transmission electron microscopy samples. \n'+
-                                                                                        'OM: optical microscopy samples')
+                                                                                        'TEM: transmission electron microscopy samples. \n' +
+                                                                                        'BF: Bright-field microscopy samples.')
     requiredName.add_argument('-i', '--imgpath', required=True, nargs='+', help='Path to the image to segment or path to the folder \n'+
                                                                                 'where the image(s) to segment is/are located.')
 
-    ap.add_argument("-m", "--model", required=False, help='Folder where the model is located. \n'+
-                                                          'The default SEM model path is: \n'+str(default_SEM_path)+'\n'+
-                                                          'The default TEM model path is: \n'+str(default_TEM_path)+'\n'+
-                                                          'The default OM model path is: \n'+str(model_seg_pns_bf_path)+'\n')
+    ap.add_argument("-m", "--model", required=False, help='Folder where the model is located.')
     ap.add_argument('-s', '--sizepixel', required=False, help='Pixel size of the image(s) to segment, in micrometers. \n'+
                                                               'If no pixel size is specified, a pixel_size_in_micrometer.txt \n'+
                                                               'file needs to be added to the image folder path. The pixel size \n'+
```

Comment:
For the "verbose" option at line 212 below. I think that verbose 2 and 3 are not used anymore in the code.
ivadomed is pretty verbose itself...
The only difference between verbose 0 and 1 is that with 1 it prints `Image /home/mhbourget/code/axondeepseg/AxonDeepSeg/models/model_seg_rat_axon-myelin_sem/data_test/image.png segmented.` after each segmented image.
I would be in favor of removing this option and default the verbose 1 behavior everywhere. What do you think?

Reply:
Hmm I'm indifferent. I like having the option available for the future. What if we set it to 1 as default, and just say 0 is quiet mode and 1 may print out additional information useful for developers ?

Reply:
The descriptions of 0 and 1 are OK as is, I was just suggesting 1 because there is only 1 line of difference between 0 and 1. We can keep both if you prefer, that's not a big issue.
However, I would remove 2 and 3 as they are not used at all anymore.

Reply:
Resolved in 0438ad28f5f4a1d